tests/unit/core/components/TestAuthenticateButton.js

```diff
@@ -65,31 +70,32 @@ describe('<AuthenticateButton />', () => {
     const handleLogIn = sinon.spy();
     const location = sinon.stub();
     const root = render({ isAuthenticated: false, handleLogIn, location });
+
     expect(root.textContent).toEqual('Log in/Sign up');
     Simulate.click(root);
-    expect(handleLogIn.calledWith(location)).toBeTruthy();
+    sinon.assert.calledWith(handleLogIn, location);
   });
 
   it('shows a log out button when authenticated', () => {
     const handleLogOut = sinon.spy();
     const root = render({ handleLogOut, isAuthenticated: true });
+
     expect(root.textContent).toEqual('Log out');
     Simulate.click(root);
-    expect(handleLogOut.called).toBeTruthy();
+    sinon.assert.called(handleLogOut);
   });
 
   it('updates the location on handleLogIn', () => {
+    const { store } = dispatchSignInActions();
     const _window = { location: '/foo' };
     const location = { pathname: '/bar', query: { q: 'wat' } };
-    const startLoginUrlStub =
-      sinon.stub(api, 'startLoginUrl').returns('https://a.m.org/login');
-    const { handleLogIn } = mapStateToProps({
-      auth: {},
-      api: { lang: 'en-GB' },
-    });
+    const startLoginUrlStub = sinon.stub(api, 'startLoginUrl').returns('https://a.m.org/login');
```

Comment:
just FYI, the reason our lines tend to be under 80 chars is because I like to use three splits in Vim. I haven't been able to convince people that our 100 line limit is too long for that :)

Reply:
ah I only have two splits in Vim 😅 

I think that 80 chars is not enough, but I am sold! Let's go for 80 chars.